### PR TITLE
Marketplace strings

### PR DESCRIPTION
**Pull request by ruthannedilig**
@fabioguarrasi

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -2,10 +2,19 @@ import App from './App.vue'
 import Vue from 'vue'
 import zaf from '@/libs/ZAFClient'
 import I18N from '@/translations'
+import store from './store'
```
Comment (fabioguarrasi) [inline src/main.js]: do you use store? if not, please remove

File: src/main.js [hunk 2]
```diff
@@ -2,10 +2,19 @@ import App from './App.vue'
...
+Vue.config.productionTip = isProduction
+
+if(isProduction) {
+  analytics.load('platform', true);
+} else {
+  analytics.load('platform_dev', false);
+}
 
 const vm = new Vue({
+  store,
```
Comment (fabioguarrasi) [inline src/main.js]: same here

File: src/store.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import Vue from 'vue'
```
Comment (fabioguarrasi) [inline src/store.js]: please remove this file if not needed

File: src/translations/en.yml [hunk 4]
```diff
@@ -12,23 +12,39 @@ parts:
   - translation:
       key: "txt.apps.select_an_address.app.short_description"
       title: "Short Description for App: Select an Address"
-      value: "View available support addresses, Pick a support address when creating a new ticket, Change the support address associated with a ticket"
-      wip: true
+      value: "Choose an email address associated with a ticket"
```
Comment (fabioguarrasi) [inline src/translations/en.yml]: is this change correct?

File: src/zaf/manifest.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "defaultLocale": "en",
+  "private": true,
+  "location": {
+    "support": {
+      "ticket_sidebar": "assets/iframe.html",
+      "new_ticket_sidebar": "assets/iframe.html"
+    }
+  },
+  "version": "2.0.1",
```
Comment (fabioguarrasi) [inline src/zaf/manifest.json]: please change with 2.0.0

File: src/translations/en.yml (hunk 4 above)
Comment (ruthannedilig) [reply src/translations/en.yml]: yes, I got it from the google docs - app page descriptions from Brent

**Comment by fabioguarrasi** [comment]
@ruthannedilig what about `VUE_APP_NAME` in `.env` file? :)

**Comment by ruthannedilig** [comment]
@zendesk/i18n can I request a review here?

File: src/translations/en.yml (hunk 4 above)
Comment (fabioguarrasi) [reply src/translations/en.yml]: got confirmation from Brent

**Pull request merged**